src/alvin0319/AmongUs/form/crew/ManifoldOpenObjectiveForm.php

```diff
@@ -78,8 +78,8 @@ public function handleResponse(Player $player, $data) : void{
 		if(strlen($this->progress) === 10){
 			if($this->progress === "0123456789"){
 				// Objective complete
-				$player->sendMessage(AmongUs::$prefix . "Objective Completed");
 				$player->getLevel()->addSound(new GenericSound($player, LevelSoundEventPacket::SOUND_LEVELUP), [$player]);
+				$player->sendTip("§aObjective Completed");
```

Comment:
This is unnecessary things. when we call `Character::completeObjective()`, that will send message to player.

Reply:
ik i removed it on my laptop
need to update it on github